packages/eds-core-react/src/components/Typography/Typography.stories.tsx

```diff
@@ -1,220 +1,80 @@
-import { Typography, TypographyProps, List } from '../..'
+import { Card, Divider, Typography, TypographyProps } from '../..'
 import styled from 'styled-components'
-import { Story, Meta } from '@storybook/react'
+import { Story, ComponentMeta } from '@storybook/react'
+import { Stack } from './../../../.storybook/components'
+import page from './Typography.docs.mdx'
+import { tokens, Typography as TypographyType } from '@equinor/eds-tokens'
 
-const Wrapper = styled.div`
-  margin: 32px;
-`
+const {
+  colors: {
+    infographic: {
+      primary__moss_green_100: { rgba: mossgreen100 },
+    },
+  },
+  typography,
+} = tokens
 
-const Grid = styled(Wrapper)`
-  display: grid;
-  grid-gap: 32px;
-  justify-items: start;
+const SBCard = styled(Card)`
+  border: 1px solid ${mossgreen100};
```

Comment:
Suggest removing the border and maybe using the `elevation` prop on `Card` instead?

Reply:
done